app/views/games/index.html.erb

```diff
@@ -14,6 +14,7 @@
         <div class="col-xs-2 text-center">Level</div>
         <div class="col-xs-2 text-center">Time</div>
         <div class="col-xs-2 text-right">Score</div>
+        <div class="col-xs-2 text-right">Chrono</div>
```

Comment:
@davidgeismar can you choose *Time* or *Chrono*, there are both same item :wink: